### PR TITLE
[#6888] test non-member group admin cannot remove users from group (4-2-stable)

### PR DESCRIPTION
**Pull request by d-w-moore**

### CODE REVIEW

File: scripts/irods/test/test_igroupadmin.py [hunk 1]
```diff
@@ -104,6 +104,10 @@ def test_group_administrator_scenarios__issue_6888(self):
 
             user_with_zone = lambda user_name,zn='': '{}#{}'.format(user_name, zn if zn else self.groupadmin.zone_name)
 
+            # Test that another groupadmin (not in the group) cannot remove a member.
+
+            self.groupadmin2.assert_icommand(['igroupadmin', 'rfg', self.group, self.groupadmin.username], 'STDERR', 'CAT_INSUFFICIENT_PRIVILEGE_LEVEL');
```
Comment (alanking) [inline scripts/irods/test/test_igroupadmin.py:109]: Remove semicolon at the end of the line?

**Review by alanking — COMMENTED** [review]
Test does as advertised in commit message :) Just one little thing

**Comment by korydraughn** [comment]
Looks good. Will approve once review comment is resolved.

File: scripts/irods/test/test_igroupadmin.py (hunk 1 above)
Comment (d-w-moore) [reply scripts/irods/test/test_igroupadmin.py]: yep.

**Review by alanking — APPROVED** [review]
Very good. Please # it

**Review by alanking — APPROVED** [review]
Looks good. Please # if test is passing

**Comment by alanking** [comment]
Oops, wrong PR. Please disregard last comment. Still approved, all the same. :)

**Review by korydraughn — APPROVED** [review]

**Review by alanking — APPROVED** [review]

**Pull request merged**